huaweicloud/services/acceptance/cbh/data_source_huaweicloud_cbh_instance_quota_test.go

```diff
@@ -0,0 +1,39 @@
+package cbh
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccDatasourceInstanceQuota_basic(t *testing.T) {
+	var (
+		rName = "data.huaweicloud_cbh_instance_quota.test"
+		dc    = acceptance.InitDataSourceCheck(rName)
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
+		},
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDatasourceInstanceQuota_basic(),
+				Check: resource.ComposeTestCheckFunc(
+					dc.CheckResourceExists(),
+					resource.TestCheckResourceAttrSet(rName, "quota"),
+					resource.TestCheckResourceAttrSet(rName, "quota_used"),
+				),
+			},
+		},
+	})
+}
+
+func testAccDatasourceInstanceQuota_basic() string {
+	return `
+data "huaweicloud_cbh_instance_quota" "test" {}
+`
+}
```

Comment:
```suggestion
const testAccDatasourceInstanceQuota_basic string = `
data "huaweicloud_cbh_instance_quota" "test" {}
`
```
If we don't have any input arguments, constant definition is better.

Reply:
done.